mandatory/4-tax.js

```diff
@@ -17,7 +20,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(productPrice) {
+  let salesTaxForProduct = (productPrice + ((productPrice * 20) / 100)).toFixed(2);
+  let showTaxAmount = "£".concat(salesTaxForProduct)
+ return showTaxAmount;
+};
```

Comment:
Comment: Interesting solution. It works?
Mine: 
function addTaxAndFormatCurrency(sales) {
return "£" + calculateSalesTax(sales).toFixed(2);
}